jbmc/regression/jbmc/context-include-exclude/test_include_all.desc

```diff
@@ -7,6 +7,5 @@ Main.class
 .* line 11 assertion at file Main.java line 11 .*: SUCCESS
 .* line 12 assertion at file Main.java line 12 .*: SUCCESS
 --
-WARNING: no body for function .*
```

Comment:
:question: is this one removed because it is useless, or does the warning now appear?

Reply:
Because it's (probably) useless. There is a test in a later commit that checks that the body of an excluded method is a stub-style "return nondet" body. Do you think it still makes sense to check for `WARNING: ...` as a negative regex everywhere?

Reply:
If there is no way this warning could be produced for any input then it can be removed, otherwise it's probably safer to keep it in just in case.

Reply:
I'm keeping it as a negative regex here now, and also added it as a negative regex in all existing tests and the new tests for exclude package and exclude absent.